python/parpe/misc.py

```diff
@@ -297,3 +297,40 @@ def simulation_to_df(mes_df, sim, result_file, start, label, observable_ids):
         mes_df.loc[row.name, 'simulation_' + label] = \
         sim[start][condition_idx][time_idx, observable_idx]
     return mes_df
+
+
+def compare_optimization_results_to_true_parameters(
+        filename: str, start_idx: str = '0'):
+    """Compare parameter estimates to true parameters. Print as table.
+
+    Used in example notebooks.
+
+    Arguments:
+        filename: Parameter estimation result file name
+        start_idx: optimizer run index/name to use
+    """
+    with h5py.File(filename, 'r') as f:
+        pscale = f['/inputData/parameters/pscaleOptimization'][:]
+        names = f['/inputData/parameters/parameterNames'][:]
+        true_parameters = f['/inputData/parameters/true_parameters'][:]
+        expectedNllh = -f['/inputData/parameters/true_llh'][:]
+
+        final_parameters = f[f'/multistarts/{start_idx}/finalParameters'][:]
+        exit_status = f[f'/multistarts/{start_idx}/exitStatus'][:]
+        final_cost = f[f'/multistarts/{start_idx}/finalCost'][:]
+
+    for i, p in enumerate(pscale):
+        if p == 2:
+            final_parameters[i] = np.power(10, final_parameters[i])
+
+    print("#  __Exp____ __Act______ __Err______ __RelErr___ __ID_______")
+    for i in range(len(true_parameters)):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using enumerate instead of iterating with range and len](https://app.codacy.com/app/dweindl/parPE/pullRequest?prid=3660516)